ADOPTERS.md

```diff
@@ -0,0 +1,10 @@
+# Who's using Kpack?
+
+### Adopters
+Below is a list of projects that have publicly adopted CNB. 
```

Comment:
Should this say "Below is a list of projects that have publicly adopted kpack." or are we trying to be more general?